db/migrate/20201107111600_create_podcast_episode_appearances.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePodcastEpisodeAppearances < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
```

Comment:
we shouldn't disable transactions during creation of new tables, if anything goes wrong, the migration could lead the DB in a corrupt state. I believe you added this so to be able to add `algorithm: concurrently` to `add_index` below, but there's no necessity to add an index on an empty table concurrently :-) `algorithm: concurrently` main purpose is not to slow down table access while the index is being created, but the table is new and empty by the time the index is added

Reply:
Yeah, I totally forgot that I dumped some data to test some things. That's the reason I added concurrently. Sorry :sweat_smile: